pkg/backup/client.go

```diff
@@ -760,7 +760,12 @@ func SendBackup(
 	req kvproto.BackupRequest,
 	respFn func(*kvproto.BackupResponse) error,
 ) error {
-	log.Info("try backup", zap.Any("backup request", req))
+	log.Info("try backup",
+		zap.Binary("StartKey", req.StartKey),
+		zap.Binary("EndKey", req.EndKey),
+		zap.Uint64("RateLimit", req.RateLimit),
+		zap.Uint32("Concurrency", req.Concurrency),
```

Comment:
```suggestion
		zap.Uint64("storeID", storeID),
```

These two are fixed and can be derived from the CLI args, so they're not worth logging (but store ID is likely more useful for debugging).